static/usage/v6/select/typeahead/angular/modal-example_component_ts.md

```diff
@@ -0,0 +1,84 @@
+```ts
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Item } from './types';
+
+@Component({
+  selector: 'app-typeahead',
+  templateUrl: 'typeahead.component.html',
+})
+export class TypeaheadComponent {
```

Comment:
~ We could add `implements OnInit` here to add type safety to the Angular life cycle methods.